src/global/AppRouter/AppRouter.tsx

```diff
@@ -5,22 +5,30 @@ import { Home } from 'pages/Home/Home';
 import { Register } from 'pages/Register/Register';
 import { Login } from 'pages/Login/Login';
 import { InitialStep } from 'pages/CreatePersona/InitialStep/InitialStep';
-import { CreateCard } from 'pages/CreatePersona/CreateCard/CreateCard';
-import { CreatePage } from 'pages/CreatePersona/CreatePage/CreatePage';
 import { Personas } from 'pages/Personas/Personas';
 import { Page } from 'pages/PersonaCreation/Page/Page';
+import { MySpots } from 'components/MySpots/MySpots';
+import { CreateSpotsStep1 } from 'pages/CreateSpot/CreateSpotsStep1';
+import { CreateSpotsCard } from 'pages/CreateSpot/CreateSpotsCard';
+import { CreateSpotsPage } from 'pages/CreateSpot/CreateSpotsPage';
+import { CreatePersonaCard } from 'pages/CreatePersona/InitialStep/CreatePersonaCard';
+import { CreatePersonaPage } from 'pages/CreatePersona/InitialStep/CreatePersonaPage';
 
 export const AppRouter: FunctionComponent = () => {
   return (
     <Switch>
       <Route path="/" exact component={Home} />
       <Route path="/scanner" exact component={() => <QrScanner onCode={res => alert(res.data)} />} />
       <Route path="/personas" exact component={Personas} />
+      <Route path="/my-spots" exact component={MySpots} />
       <Route path="/login" exact component={Login} />
+      <Route path="/creation/step/1/entity/spot" exact component={CreateSpotsStep1} />
```

Comment:
why urls are handled differently than ones used for create persona?

Reply:
Patryk with Mateusz they told me to set this link